docs/outlook/outlook-addin-design.md

```diff
@@ -14,41 +14,41 @@ The following high-level guidelines will help you design and build a compelling
 
```

Comment:
Not relevant for this PR, but I'm pretty sure we have a version of this article for non-Outlook add-ins. It's probably worth merging them.